pipeline/clean/opuscleaner/configs/remove_frequent_patterns.txt

```diff
@@ -0,0 +1,9 @@
+# Starting text in a lot of rows in OpenSubtitles, not everytime in both source and targets
```

Comment:
Suggestion: This first comment doesn't make sense to me. It would be nice to re-phrase it using a more complete sentence.

The same goes for "Text in Czech Bible translations that is not in the source"